examples/aishell/NST/README.md

```diff
@@ -0,0 +1,139 @@
+```
+
+### Noisy student interations
+
+After finishing the initial fully supervised baseline, we now have the pseudo-label data list which is `wenet_1khr_nst0.list` if you follow the guideline. We will use it as the `pseudo_data` in the training step and the `pseudo-label` for next NST iteration will be generated.
+
+Here is an example command:
+
+```bash
+bash run_nst.sh --dir exp/conformer_nst1 --supervised_data_list data_aishell.list --pseudo_data_list wenet_1khr_nst0.list  --enable_nst 1 --job_num 0 --hypo_name hypothesis_nst1.txt --untar_dir data/train/wenet_1khr_untar_nst1/ --tar_dir data/train/wenet_1khr_tar_nst1/ --out_data_list data/train/wenet_1khr_nst1.list 
+```
+Most of the arguments are same as the initial teacher training, here we add extra argument `pseudo_data_list` for path of pseudo data list. The `enbale_nst` must be set to 1 if you want to train with pseudo data. The index for `hypo_name` and `tar_dir` need to be changed if you don't want to overlap the previous generated data.
+The `output_data_list` can be used as the input of `pseudo_data_list` for next NST itearion.
+
+Full arguments are listed below, you can check the `run_nst.sh` for more information about each stage and their arguments:
+```bash
+bash run_nst.sh --stage 1 --stop-stage 8 --dir exp/conformer_nst1 --supervised_data_list data_aishell.list --pseudo_data_list wenet_1khr_nst0  --enable_nst 1 --num_split 1 --dir_split wenet_split_60_test/ --job_num 0 --hypo_name hypothesis_nst1.txt --label 0 --wav_dir data/train/wenet_1k_untar/ --cer_hypo_dir wenet_cer_hypo --cer_label_dir wenet_cer_label --label_file label.txt --cer_hypo_threshold 10 --speak_rate_threshold 0 --utter_time_file utter_time.json --untar_dir data/train/wenet_1khr_untar_nst1/ --tar_dir data/train/wenet_1khr_tar_nst1/ --out_data_list data/train/wenet_1khr_nst1.list 
+```
+
+## Performance Record
+
+### Supervised baseline and standard NST
+* Without filter strategy, first iteration
+* Feature info: using FBANK feature, dither, cmvn, online speed perturb
+* Training info: lr 0.002, batch size 32, 8 gpu, acc_grad 4, 240 epochs, dither 0.1
+* Decoding info: ctc_weight 0.3, average_num 30
+
+
+| Supervised               | Unsupervised | Test CER |
+|--------------------------|--------------|----------|
+| AISHELL-1 Only           | ----         | 4.85     |
+| AISHELL-1+WenetSpeech    | ----         | 3.54     |
+| AISHELL-1+AISHELL-2      | ----         | 1.01     |
+| AISHELL-1 (standard NST) | WenetSpeech  | 5.52     |
```

Comment:
According to your .yaml configuration, this is a **non-streaming model** and  `4.85` is obtained from **attention restoring**. 

I do think we'd better mention those Infos directly in README.